curriculum/challenges/english/25-front-end-development/workshop-storytelling-app/671fa7ed8364b130b5c55d2e.md

```diff
@@ -0,0 +1,142 @@
+---
+id: 671fa7ed8364b130b5c55d2e
+title: Step 8
+challengeType: 0
+dashedName: step-8
+---
+
+# --description--
+
+Select the `#result` paragraph and store it in a variable called `resultParagraph`. You will use this variable to display the selected story.
+
+# --hints--
+
+Your `resultParagraph` variable should have the value of `document.getElementById("result")`.
+
+```js
+assert.deepEqual(resultParagraph, document.getElementById("result"));
+```
```

Comment:
Same here.
querySelector is also a valid option.
So the hint needs to be updated to account for that

Reply:
Done!